types/koa-generic-session/index.d.ts

```diff
@@ -50,5 +51,13 @@ declare namespace koaSession {
     const MemoryStore: SessionStore;
 }
 
+declare module 'koa' {
+    interface Context {
+        session: koaSession.Session|null;
+        sessionSave: boolean|null;
+        regenerateSession(): Generator;
```

Comment:
`Generator` defined in `lib.es6.ts` from TypeScript starndard library.
This type definition already setting up to import `es6`  lib, but got error 🤔 https://github.com/Ragg-/DefinitelyTyped/blob/8117819ce704fe73c3ac9f0db5cb9dfbf2b70f01/types/koa-generic-session/tsconfig.json#L5